File: examples/java/java-booking-02_introduce_parameter_object/src/main/java/tv/codely/booking/DateRange.java
package tv.codely.booking;

import java.time.LocalDateTime;

public final class DateRange {
    private final LocalDateTime startDate;
    private final LocalDateTime endDate;

    public DateRange(LocalDateTime startDate, LocalDateTime endDate) {
        this.startDate = startDate;
        this.endDate   = endDate;
    }

    public LocalDateTime startDate() {
        return startDate;
    }

    public LocalDateTime endDate() {
        return endDate;
    }
}

File: examples/csharp/csharp-booking-02_introduce_parameter_object/src/Booking/DateRange.cs
using System;

namespace CodelyTv.Booking
{
    public sealed class DateRange
    {
        public DateTime StartDate { get; }
        public DateTime EndDate { get; }

        public DateRange(DateTime startDate, DateTime endDate)
        {
            this.StartDate = startDate;
            this.EndDate = endDate;
        }
    }
}
